sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -735,12 +735,16 @@
 
   /**
    * @since 2.12
+   * @deprecated since 6.2 all coverage reports are merged in the same measures 
    */
+  @Deprecated
   public static final String IT_COVERAGE_KEY = "it_coverage";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)